sdk/Directory.Build.props

```diff
@@ -1,13 +1,3 @@
 ﻿<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <PropertyGroup>
-    <IsMgmtProject Condition="'$(IsMgmtProject)' == '' and $(MSBuildProjectName.Contains('.Management.'))">true</IsMgmtProject>
-    <IsDataPlaneProject Condition="'$(IsDataPlaneProject)' == '' and $(MSBuildProjectName.StartsWith('Microsoft.Azure.')) and '$(IsMgmtProject)' != 'true'">true</IsDataPlaneProject>
-  </PropertyGroup>  
-  <ItemGroup Condition="'$(IsTestProject)' == 'true' or '$(IsPerfProject)' == 'true' or '$(IsStressProject)' == 'true'">
-    <AssemblyAttribute Include="System.Reflection.AssemblyMetadataAttribute">
-      <_Parameter1>SourcePath</_Parameter1>
-      <_Parameter2>$(MSBuildProjectDirectory)</_Parameter2>
-    </AssemblyAttribute>
-  </ItemGroup>
   <Import Project="..\Directory.Build.props" />
```

Comment:
will this file be removed in the future since all it does now is forward?

Reply:
Yes I was planning to remove it once I proved everything works.